panel/widgets/player.py

```diff
@@ -11,7 +11,7 @@
 
 class PlayerBase(Widget):
 
-    interval = param.Integer(default=500, doc="Interval between updates")
+    interval = param.Integer(default=500, doc="Interval between updates, in milliseconds. Default is 500, i.e. two updates per second")
```

Comment:
```suggestion
    interval = param.Integer(default=500, doc="""
        Interval between updates, in milliseconds. Default is 500, i.e. 
        two updates per second.""")
```

Reply:
Thinking about this from API design perspective, this information could be encoded in the type of the property, i.e. have `param.Milliseconds(...)`, which would be, at least at first, a simple alias for `param.Integer` (well, I suppose non-negative integer, as most likely we don't want negative intervals to be allowed). This then would allow to push the API even further, e.g. allowing string values like `1s` that would be parsed and normalized to the base type and units.

Reply:
interesting idea. you could probably generalize that a bit further by making it a generic [timedelta](https://docs.python.org/3/library/datetime.html#timedelta-objects) type with a "default units" option or something like that.

Reply:
Unit support in Param is a longstanding issue, and is something that is orthogonal to the Parameter type in general. A Parameter _instance_ (not necessarily class) should be able to declare its type, e.g. using [pint](https://pint.readthedocs.io), and then if someone passes a value with a compatible type, it will be converted into the desired type. We had a partial implementation at some point, but I don't see it in the PRs, so that might have predated this project being on Github.  Still something worth doing!